scipy/optimize/__init__.py

```diff
@@ -110,7 +110,8 @@
 .. autosummary::
    :toctree: generated/
 
-   fixed_point - Single-variable fixed-point solver
+   fixpoint - Single-variable fixed-point solver
+   fixed_point - Steffensen's method for fixed points
```

Comment:
The problem here is that the signature of fixed_point was fairly
specific for the steffensen/aitken naive iteration, and changing it
breaks backward compat.
.
The actual difference as it is in this PR is that fixed_point is deprecated.